apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -171,6 +232,7 @@
                                     class="ui primary button"/>
                             </div>
                     </form>
+                    <script>onload();</script>
```

Comment:
This will not ensure that the `onload()` is called always after the page has been loaded. The best practice is to use event handlers instead.

ref: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event

Also, shall we check whether we can use `ready` event instead of `load` event? the `load` event is fired only after the all dependencies such as images etc are loaded, while `ready` event is fired as soon as the HTML is loaded.

Ref: https://www.geeksforgeeks.org/difference-between-body-onload-and-document-ready-function/